### ISSUE TITLE
Flaw in BlackMATE theme for evolution mail client

### ISSUE THREAD

**Issue by pkzc** [mate-desktop/mate-themes#311]
#### Expected behaviour
All text in all windows, panels, etc. are readable in evolution.

#### Actual behaviour
When using the BlackMATE theme, the Description field on the General tab of an evolution calendar entry is shown in white letters on a practically white background, and is unreadable.

Line 24 in /usr/share/themes/BlackMATE/gtk-3.0/gtk-main.css of mate-themes-3.22.24-2.fc39.noarch is this:
@define-color view_color #ECECEC;
This is almost white, changing it to  #232323 or similar solves the problem in evolution. It may cause regression somehere else, but I haven't noticed any so far.

#### Steps to reproduce the behaviour
Double click on a calendar entry to open it.

#### MATE general version
mate-desktop-1.26.2-1.fc39.x86_64

#### Package version
mate-themes-3.22.24-2.fc39.noarch

#### Linux Distribution
Fedora Linux 39

#### Link to bugreport of your Distribution (requirement)
https://gitlab.gnome.org/GNOME/evolution/-/issues/2649

**Comment by lukefromdc** [comment]
Unfortunately I cannot work on that, because I have no email accounts that can be accessed other
than by webmail. Evolution won't get past the setup screen without one.

**Comment by lukefromdc** [comment]
Thus I will have to leave this for someone else on the team who can install and run Evolution.
I can install it, but I cannot run it past the setup screen without an email account accessable
by a dedicated email program. Many non-ad supported and free email accounts (the only
kind I have) are webmail-only.

**Comment by pkzc** [comment]
I think evolution is just fine with that change. The real question seems to me  whether it messes something up in some other application.

**Comment by lukefromdc** [comment]
That change I can test and will, thanks

**Comment by lukefromdc** [comment]
That created a major problem in a  test with gtk3-widget-factory

![BlackMATE_test](https://github.com/mate-desktop/mate-themes/assets/14036792/1dc7d9a6-1233-4c9e-80ac-5c8749f53e43)

A test: Do you also get the same issue in adwaita-dark? If you do, the issue is in evolution. If not, we need to identify
the offending widget by name or name of an ancestor so as to restrict the change to evolution only.

**Comment by pkzc** [comment]
No, Adwaita-dark lends a  readable look to evolution all over. That was why I reported to BlackMATE.

But then  isn't it a problem with gtk3-widget-factory? It would look quite a bit strange to me if we had two glaring white patches  in your picture, and kind of much more natural  the present dark background with white or light gray letters.

**Comment by lukefromdc** [comment]
Gtk3-widget-factory is intended for such jobs as testing themes and learning how GTK's widgets work.
To find a "widget chain" or better an application-specific widget name, you need to open evolution with
gtk-inspector.  You can enable it in dconf-editor, then shift-control-D will bring it up.

We REALLY need someone on the team who can run Evolution to work on this!

**Comment by pkzc** [comment]
> That created a major problem in a test with gtk3-widget-factory
> 
> ![BlackMATE_test](https://github.com/mate-desktop/mate-themes/assets/14036792/1dc7d9a6-1233-4c9e-80ac-5c8749f53e43)
> 
> A test: Do you also get the same issue in adwaita-dark? If you do, the issue is in evolution. If not, we need to identify the offending widget by name or name of an ancestor so as to restrict the change to evolution only.

I had the same problem somwhere else outside of evolution and I think I found the culprit:  these might be the correct settings in BlackMATE gtk-main.css:
```
@define-color theme_main_color #ECECEC; /* #000000;*/
@define-color view_color       #232323; /* #ECECEC */
```
The original values are shown in comments, i.e. they were just inverted.

**Comment by lukefromdc** [comment]
Reversing the view color would be a major change in the appearance of such apps as pluma and probably
caja as well

**Comment by pkzc** [comment]
I don't use pluma so don't know, but with caja, I haven't observed any adverse effects so far, and as far as the MATE gsmartcontrol GUI is concerned, it looks much better after the tweak (in my opinion, of course).

Here is its look in the original BlackMATE, an irritating radiation in the middle of a dark desktop:

<img width="918" height="687" alt="Image" src="https://github.com/user-attachments/assets/459e315b-5137-43c2-aeca-0bdb74469176" />

And here is after the colors have been reversed, a soothing experience:

<img width="928" height="697" alt="Image" src="https://github.com/user-attachments/assets/8b233899-7104-46a7-9bd5-83463afc9ea3" />

**Comment by lukefromdc** [comment]
The light views are probably intentional in this theme, which is copied from an earlier GNOME theme.
Caja and Pluma look different in BlackMATE (caja view dark, pluma view light), but they have 
always been this way. Not changing this without a lot of feedback from others as those used to
it as-is may have an issue with it.

For a full-dark theme (BlackMATE like my own UbuntuStudio-Legacy theme is a mixed theme), try
Bluementa-dark or the more generic Adwaita-dark theme. Note that the latter was not written for 
MATE but we try to provide sane defaults for themes that do not explicitly support MATE

**Comment by pkzc** [comment]
> pluma view light

OK, I didn't know this, not even suspected. Then, perhaps, it would make sense to close this.
As for the Bluementa-dark, it must be something new, I will try to find it, thanks for the suggestion.

**Comment by lukefromdc** [comment]
Evolution (which I can now run without a supported email account)  fixed something, and no longer shows white on white text. Evolution made some change that we now have white text on a dark background, so we can close this as the orignal problem seems to be gone.

If a specific app is jarring, get a widget name or widget chain unique to that app, and you can fix it locally in ~/.config/gtk-3.0/gtk.css
